lib/services/android-project-service.ts

```diff
@@ -298,6 +298,10 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 		return this.$fs.exists(path.join(this.platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME));
 	}
 
+	public isPlatformBuilt(projectRoot: string): IFuture<boolean> {
+		return this.$fs.exists(path.join(projectRoot, "build", "outputs", "apk"));
```

Comment:
Actually this is enough to work and it solves the issue. Of course it won't work if someone removes everything from the build folder explicitly (The CLI always removes the whole folder). We could add a check whether there are apk files, but who guarantees that those are the apk files generated by us? Maybe the user placed his own files there? Do you think that we should check this too?